cmd/skipper/main.go

```diff
@@ -569,12 +569,6 @@ func main() {
 		eus = strings.Split(etcdUrls, ",")
 	}
 
-	clsic, err := parseDurationFlag(closeIdleConnsPeriod)
```

Comment:
if we want to stay consistent, then we should remove the parseDurationFlag usage from the breakerflags.go and ratelimiterflags.go, as well, and delete the parseDurationFlag function.